tests/src/test/scala/whisk/core/database/test/DbUtils.scala

```diff
@@ -128,6 +128,28 @@ trait DbUtils extends Assertions {
     assert(success.isSuccess, "wait aborted")
   }
 
+  /**
+   * Wait on a view to update with documents added(don't specify the namespace). This uses retry above,
+   * where the step performs a direct db query to retrieve the view and check the count
+   * matches the given value.
+   */
+  def waitOnView[Au](db: ArtifactStore[Au], count: Int, view: View)(implicit context: ExecutionContext,
```

Comment:
Given this we can then refactor `ArtifactStoreSubjectQueryBehaviors` to drop `waitOnBlacklistView` and `blacklistCount` methods

Reply:
updated